portafly/src/utils/index.ts

```diff
@@ -19,4 +20,37 @@ const combineReducers = (reducerSlices: Record<string, any>) => (
     prevState
   ))
 
-export { createReducer, combineReducers }
+// # Auth
+// Token is stored in JSON and not string since in the future we will be working with JWT
+const getToken = () => JSON.parse(<string>localStorage.getItem('token'))
```

Comment:
Cast is redundant here. Also I would define `AuthToken` in this file instead:
```suggestion
const getToken: AuthToken = () => JSON.parse(localStorage.getItem('token'))
```

Reply:
I've changed the fn signature, but still needs the cast since `JSON.parse` can't parse `null`. A more robust way would be having a fn like in Porta UI. `SafeParse`

Reply:
Ah you're right, I don't know why I thought that.

A thought tho: I think this kind of casting is deceitful, because it is used only to make TS happy. `getItem` _can return_ `null` and therefore it is a case to be considered. It's not because the code won't work in this particular case (it is 100% good) but because we're being dishonest with TS and we should always avoid that.
```js
// we could return an empty JWT / Object when the token is null
JSON.parse(localStorage.getItem('token') || '{}')

// or we could check
item = localStorage.getItem('token')
return item ? JSON.parse(item) : {}

// or we could do a "real" casting
JSON.parse(String(localStorage.getItem('token'))
```

Reply:
Yeah, it's quite a double edge sword... actually more than that, since sometimes we just can't be sure

Reply:
Still, we are not sure how the flow will be, so maybe I'll go for checking or do a "real casting"